server/controllers/comment.controller.js

```diff
@@ -70,6 +67,26 @@ class CommentController {
       return Response(res, 201, 'thread comment added', response);
     }
   }
+
+  /**
+   * Like comment controller
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} response
+   */
+  static async likeComment(req, res) {
+    const { id: userid } = req.user;
+    const { commentid } = req.params;
+
+    const likeComment = await commentlikes.create({
+      userid,
+      commentid,
+    });
+    if (likeComment) {
+      const like = { like: likeComment };
+      return Response(res, 201, 'Comment liked successfully', like);
+    }
```

Comment:
What happen's when (!likeComment) happens. If there's no situation to cater for (!likeComment), then the if block on line 85 should not be there at all.

Reply:
Implemented this...thanks